### DISCUSSION TITLE
How to increase drug concentration in kidney tissue in PBPK model？

### DISCUSSION THREAD

**Discussion by mixieer358** [Open-Systems-Pharmacology/Forum#1446] [Questions & Problems]
I'd like to know whether  PKSIM can be used to increase the concentration of drugs in the Kidney？or the concentration of the free drug. Thanks！

**Comment by StephanSchaller** [comment]
Dear @mixieer358 , can you elaborate what you mean with:

> increase the concentration of drugs in the Kidney

Thanks,
Stephan

**Reply by mixieer358** [reply]
Dear @StephanSchaller

I'm sorry for the lack of clarity. 
Renal OCT2 on the basolateral tubule membrane works to control metformin disposition. Is metformin uptake significantly reduced if I increase OCT2 inhibitor concentration on the proximal renal tubular base in PBPK modeling? I want to implement this idea with PKSIM but I don't know how to do it.

Thanks,
mixieer358

**Reply by StephanSchaller** [reply]
Hi  @mixieer358, there is a published Metformin Model including OCT2: https://github.com/Open-Systems-Pharmacology/Metformin-Model
